SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-cloud.md

```diff
@@ -99,7 +99,8 @@ To launch a cloud deployment:
 1. Enter your project name (see [Set up your SpatialOS project name](#step-1-set-up-your-spatialos-project-name)). 
 1. In the **Assembly Name** field, enter the name you gave your assembly in the [previous step](#step-3-upload-your-workers).
 1. In the **Deployment Name** field, enter a name for your deployment. This labels the deployment in the [Console]({{urlRoot}}/content/glossary#console).
-1. Leave the Snapshot File field as it is. In the **Launch Config File** field, enter the path to `one_worker_test.json` (including the file name).
+1. Add the absolute path to the **Launch Config File** `~yourprojectname/spatial/one_worker_test.json`  
```

Comment:
I think the way we usually do this is `/<ProjectRoot>/spatial/one_worker_test.json`